### ISSUE TITLE
Inline event handlers in markup cause errors when used without quotes

### ISSUE THREAD

**Issue by alexprey** [alexprey/sveltedoc-parser#28]
The following markup cause exception while parsing component

```html
<button on:click={() => dispatch('notify2')}>
    CLICK!
</button>

<button on:click={function () { dispatch('notify2'); }}>
    CLICK!
</button>
```

**Comment by alexprey** [comment]
The cause of this issue is a not supported HTML format for statements like following:
```
<tag attribute={() => { some another actions that mighty can be helpfully; } />
```

**Comment by alexprey** [comment]
Actually the issue caused in base library that used for HTML parsing, so I'm create a PR in fork version of htmlparser2 package that still in progress https://github.com/alexprey/htmlparser2/pull/1

**Comment by alexprey** [comment]
Fixed and published with new package of version 3.0.3
